.github/actions/setup/action.yml

```diff
@@ -121,4 +121,4 @@ runs:
 
     - name: 'Install vcpkg'
       shell: bash
-      run: ./Toolchain/BuildVcpkg.sh
+      run: python3 ./Toolchain/BuildVcpkg.py
```

Comment:
I think we have a shebang in there, can we drop the `python3` here and in `libjs-test262.yml`?

Reply:
The executable bit is set though?